src/dags/gob.py

```diff
@@ -121,8 +137,10 @@ def create_gob_dag(
         f"{dag_id}_{dataset_table_id}",
         default_args=new_default_args,
         schedule_interval=schedule_interval,
-        tags=["gob"] + [gob_dataset_id],
+        tags=["gob"] + [gob_dataset_id] + list(schedule_labels or []),
```

Comment:
... then you don't need the list call here, viz.

```suggestion
tags=["gob"] + [gob_dataset_id] + schedule_labels,
```